tests/system/aiplatform/test_prediction_cpr.py

```diff
@@ -0,0 +1,108 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import datetime
+import json
+import logging
+import os
+import subprocess
+
+from tests.system.aiplatform.test_resources.cpr_user_code.predictor import (
+    SklearnPredictor,
+)
+
+from google.cloud import aiplatform
+from google.cloud.aiplatform import models
+from google.cloud.aiplatform.prediction import LocalModel
+
+from tests.system.aiplatform import e2e_base
+
+_TIMESTAMP = f"{datetime.datetime.now().strftime('%Y%m%d_%H%M%S')}"
+_IMAGE_URI = f"gcr.io/ucaip-sample-tests/prediction-cpr/sklearn:{_TIMESTAMP}"
+_DIR_NAME = os.path.dirname(os.path.abspath(__file__))
+_USER_CODE_DIR = os.path.join(_DIR_NAME, "test_resources/cpr_user_code")
+_REQUIREMENTS_FILE = "requirements.txt"
+_DIR_NAME = os.path.dirname(os.path.abspath(__file__))
+_LOCAL_MODEL_DIR = os.path.join(_DIR_NAME, "test_resources/cpr_model")
+_ARTIFACT_URI = "gs://cloud-aiplatform-us-central1/vertex-ai/prediction-cpr/sklearn"
+_PREDICTION_INPUT = [[4.6, 3.1, 1.5, 0.2]]
+
+
```

Comment:
```suggestion
@pytest.mark.usefixtures("delete_staging_bucket", "tear_down_resources")
```

Reply:
Otherwise your created resources will pile up indefinitely until hitting quota

Reply:
https://github.com/googleapis/python-aiplatform/blob/custom-prediction-routine/tests/system/aiplatform/test_prediction_cpr.py#L46

We don't create new buckets in this test so we don't need to have `delete_staging_bucket`.